modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/string.ts

```diff
@@ -12,6 +12,24 @@
  * details.
  */
 
+const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
+
+type Label = {
+	[key: string]: string;
+};
+
+/**
+ * Get the label according to the locale
+ */
+
+export function getLocale(
```

Comment:
hey @alinedoleron, I think we can rename this function to `getLocalizableLabel`, it makes more sense to me, what do you think?